language/tools/cost-synthesis/src/main.rs

```diff
@@ -157,49 +158,91 @@ fn stack_instructions(options: &Opt) {
     ];
 
     let mut account = Account::new();
-    with_loaded_vm! (3, options.num_iters as usize, account => vm, loaded_module, module_cache);
+
```

Comment:
@tzakian this is the macro inlined. It actually simplified the code a bit too and it seems the right thing to do given no one else was using the macro. I suspect that was useful when the test-generation was using it, but that is not the case any longer